lib/org/openpsa/invoices/handler/invoice/view.php

```diff
@@ -84,7 +84,7 @@ private function populate_toolbar()
             $buttons[] = $this->build_button('mark_paid', 'check');
         }
 
-        if ($this->_config->get('invoice_pdfbuilder_class')) {
+        if ($this->_config->get('invoice_pdfbuilder_class_bill')) {
```

Comment:
how's about not changing this (so that there's no compatibility issue with external repos), but just appending  `_reminder` for the new functionality?